src/pages/HomePage.tsx

```diff
@@ -0,0 +1,15 @@
+import { useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import { Routes } from "../routes/router";
+
+function HomePage() {
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    navigate(Routes.SIGNUP);
+  }, [navigate]);
+
+  return null;
+}
+
```

Comment:
I think you can use your dashboard as your home page, and protectedRoute will do the work of redirecting depending on whether the user is authenticated or not.